### ISSUE TITLE
Add a lead paragraph to explain clustering.

### ISSUE THREAD

**Issue by ShawnMilo** [nats-io/nats-site#183]
The [clustering documentation](http://nats.io/documentation/server/gnatsd-cluster/) contains everything necessary to set up a cluster. 

However, it requires reading about half the page to understand the basics of how it works. If the first paragraph contained something like the following, it would help people get started more quickly.

----

In addition to a port for listening for clients, `gnatsd` can listen on a "cluster" URL (the `-cluster` option). Additional `gnatsd` servers can then add that URL to their `-routes` argument to join the cluster.

Examples:

Without clustering:

```bash
gnatsd -p 4222
```

Simple clustering:

```bash
# server A on 10.10.0.1
gnatsd -p 4222 -cluster nats://10.10.0.1:5222

# server B on 10.10.0.2
gnatsd -p 4222 -routes nats://10.10.0.1:5222
```

Server B may also serve a "cluster" URL, and Server A may add that to its routes. However it is not required to do this; once a server connects to the cluster it becomes aware of all other servers in that cluster.

```bash
# server A on 10.10.0.1
gnatsd -p 4222 -cluster nats://10.10.0.1:5222 -routes nats://10.10.0.2:5222

# server B on 10.10.0.2
gnatsd -p 4222 -cluster nats://10.10.0.2:5222 -routes nats://10.10.0.1:5222 
```
Clients connecting to any server in the cluster will remain connected to the cluster even if the server it originally connected to is taken down, as long as at least a single server remains.

**Comment by ShawnMilo** [comment]
`taken(hint)`

**Comment by mcqueary** [comment]
Closed by #185
